doc/gov/chapters/chapter09.md

```diff
@@ -1,135 +1,157 @@
 # 9. Adoption
-<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [9.1 Introduction](#9.1)
-* [9.2 Transition Plan](#9.2)
-  * [9.2.1 Conformance Levels](#9.2.1)
-  * [9.2.2 Exception Types](#9.2.2)
-  * [9.2.3 Transition Framework](#9.2.3)
-* [9.3 Adoption Strategy](#9.3)
-  * [9.3.1 Expectations from Operators](#9.3.1)
-  * [9.3.2 Expectations from Vendors](#9.3.2)
-  * [9.3.3 Expectations from Industry](#9.3.3)
-* [9.4 Adoption Roadmaps](#9.4)
+  * [9.1.1 Background](#9.1.1)
+  * [9.1.2 Field Trial Purpose](#9.1.2)
+  * [9.1.3 Purpose of this Document Section](#9.1.3)  
+* [9.2 Adoption Strategy](#9.3)
+  * [9.2.1 Expectations from Operators](#9.3.1)
+  * [9.2.2 Expectations from Vendors](#9.3.2)
+  * [9.2.3 Expectations from Industry](#9.3.3)
+* [9.3 Adoption Roadmaps](#9.4)
+* [9.4 Transition Plan](#9.2)
+  * [9.4.1 Conformance Levels](#9.2.1)
+  * [9.4.2 Exception Types](#9.2.2)
+  * [9.4.3 Transition Framework](#9.2.3)
 * [9.5 CNTT Field Trial Approach] (#9.5) 
+
 <a name="9.1"></a>
 ## 9.1 Introduction
+It is vitally important for the success of the CNTT mission to have as many working CNTT compliant solutions, including infrastructure and VNFs/CNFs designs from the vendor community as possible. Obviously, there will be solutions that will not be able to be fully aligned with CNTT requirements, however, the intention is to make the CNTT reference architectures and implementations map to the real world so to make compliance attractive to the broader community. Therefore, a transition plan, an adoption strategy, and adoption roadmap needs to be agreed on within the CNTT community. The intention of this document is to detail the strategy for broader adoption in the larger telecom ecosystem. 
```

Comment:
Typo: "... and VNFs/CNFs designs ..." ---> "... and VNF/CNF designs ..."

Reply:
fixed